php.ini-development

```diff
@@ -356,6 +356,12 @@ disable_classes =
 ; http://php.net/realpath-cache-ttl
 ;realpath_cache_ttl = 120
 
+; Allows user to disable the "stat cache." By default php remembers the
```

Comment:
`stat cache."` → `stat cache".`

Reply:
Chicago Elements of Style would disagree, but ok.